packages/s3/s3.server.js

```diff
@@ -122,11 +122,15 @@ FS.Store.S3 = function(name, options) {
       return fileObj.collectionName + '/' + fileObj._id + '-' + (filenameInStore || filename);
     },
     createReadStream: function(fileKey, options) {
-
-      return S3.createReadStream({
+      var params = {
         Bucket: bucket,
         Key: folder + fileKey
-      });
+      };
+
+      if (options && options.start && options.end)
```

Comment:
This condition will fail when start === 0 (unless start and end come in as strings, I can't remember)